src/main/xar-resources/data/xquery/xquery.xml

```diff
@@ -147,14 +147,9 @@
                     <para> <literal>map:merge#2</literal> </para>
                 </listitem>
             </itemizedlist>
-
-            <para>eXist-db supports all standard serialization parameters except the
-                following:</para>
-            <itemizedlist>
-                <listitem>
-                    <para> <literal>item-separator</literal> </para>
-                </listitem>
-            </itemizedlist>
+                <para>eXist-db supports all standard serialization parameters, but some bugs remain in the implementation, as reported by XQuery Test Suite (XQTS). For the internal tests that pass, see the<link condition="_blank"
```

Comment:
A space is needed between `the` and `<link` in `the<link`.

Reply:
yes sorry about that

Reply:
should be good now

Reply:
Yes, thanks!